subsys/bluetooth/audio/has_internal.h

```diff
@@ -44,6 +44,13 @@
 #define BT_HAS_CHANGE_ID_PRESET_AVAILABLE   0x02
 #define BT_HAS_CHANGE_ID_PRESET_UNAVAILABLE 0x03
 
+#define BT_HAS_PRESET_NAME_MIN 1
+#define BT_HAS_PRESET_NAME_MAX 40
+
+#define BT_HAS_PRESET_INDEX_NONE 0x00
+#define BT_HAS_PRESET_INDEX_FIRST 0x01
+#define BT_HAS_PRESET_INDEX_LAST 0xFF
```

Comment:
I think these values are better placed in the public header file, as they are restrictions to application-provided arguments.